commercetools-sdk-base/src/main/resources/META-INF/services/javax.money.spi.ServiceProvider

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2012, 2015, Credit Suisse (Anatole Tresch), Werner Keil and others by the @author tag.
```

Comment:
actually it should be removed 👍  it's a leftover from wraping javax money to including it in sdk-base